elasticlib/Elastic.java

```diff
@@ -33,10 +33,40 @@ public static class ElasticNotification {
     @JsonProperty("description")
     private String description;
 
-    public ElasticNotification(NotificationLevel level, String title, String description) {
+    @JsonProperty("displayTime")
+    private int displayTimeMillis;
+
+    @JsonProperty("width")
+    private double width;
+
+    @JsonProperty("height")
+    private Double height;
+
+    public ElasticNotification(
+        NotificationLevel level,
+        String title,
+        String description,
+        int displayTimeMillis,
+        double width,
+        Double height) {
       this.level = level;
       this.title = title;
+      this.displayTimeMillis = displayTimeMillis;
       this.description = description;
+      this.height = height;
+      this.width = width;
+    }
+
+    public ElasticNotification(NotificationLevel level, String title, String description) {
+      this(level, title, description, 3000, 350, null);
+    }
+
+    public ElasticNotification(NotificationLevel level, String title, String description, int displayTimeMillis) {
+      this(level, title, description, displayTimeMillis, 350, null);
+    }
+
+    public ElasticNotification(NotificationLevel level, String title, String description, double width, Double height) {
```

Comment:
`Double height` should be changed to `double height`

Reply:
How would I pass it as null then?